packaging/linux/flatpak/dev.lizardbyte.sunshine.yml

```diff
@@ -5,14 +5,15 @@ sdk: org.freedesktop.Sdk
 command: sunshine
 separate-locales: false
 finish-args:
-  - --share=ipc
   - --socket=x11
   - --socket=wayland
   - --socket=pulseaudio
+  - --share=ipc
   - --share=network
   - --device=all
   - --persist=.config/sunshine
   - --system-talk-name=org.freedesktop.Avahi
+  - --talk-name=org.freedesktop.Flatpak
   - --env=PULSE_PROP_media.category=Manager
```

Comment:
Is the order of these args important? If not can we put them in alphabetical order?

Reply:
It is not important. That sounds good to me.